### PR TITLE
Add <Space> instructions for sub-tasks

### PR DESCRIPTION
**Pull request by mulholio**
I couldn't work out how to complete sub-tasks for a second. Found it and thought it would be helpful to add to the help modal!

### CODE REVIEW

**Comment by smallhadroncollider** [comment]
Could the wording maybe be:

> Mark subtask as (in)complete

File: src/UI/Draw/Modal/Help.hs [hunk 1]
```diff
@@ -34,7 +34,7 @@ descriptions =
     , ([A.ClearDate], "Removes due date")
     , ([A.MoveUp, A.MoveDown], "Shift task down / up")
     , ([A.MoveLeft, A.MoveRight], "Shift task left / right")
-    , ([A.Complete], "Move task to last list and remove any due dates")
+    , ([A.Complete], "Move task to last list and remove any due dates / Do/undo sub-task")
```
Comment (mulholio) [inline src/UI/Draw/Modal/Help.hs]:
```suggestion
    , ([A.Complete], "Move task to last list and remove any due dates / Mark subtask as (in)complete")
```

**Comment by smallhadroncollider** [comment]
Thanks! I'm going to update it to merge into `develop`, which might lead to all sorts of messages, but hopefully shouldn't cause any issues.

**Comment by smallhadroncollider** [comment]
I completely messed up merging this and it blitzed my commit history in ways that scared me. So I reverted and made the change myself. I made it with you as the author though, so hopefully you'll still appear as a contributor.

**Comment by mulholio** [comment]
No worries just wanted to make this better!

**Pull request merged**